<launch>
  <param name="use_gui" value="true"/>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find dingo_dock)/rviz/navigation.rviz" />
  
  <node name="pointcloud_to_laserscan" pkg="pointcloud_to_laserscan" type="pointcloud_to_laserscan_node">
  
    <remap from="/cloud_in" to="/velodyne_points"/>
    <remap from="/scan" to="/front/scan"/>
    
    <param name="min_height" value="0.0"/>
    <param name="max_height" value="0.5"/>
    <param name="range_min" value="0.0"/>
    <param name="range_max" value="100"/>
    
  </node>  
</launch>
